frontend/src/reducers/windowHandler.js

```diff
@@ -108,7 +108,7 @@ const initialModalState = {
  * - inlineTab keys ${windowId}_{$tabId}_${rowId}
  */
 export const initialState = {
-  connectionError: false,
+  errorType: '',
```

Comment:
move this to `appHandler.js` because the connection error is more about the whole application and not about just the window itself.

also i would rename it to `connectionErrorType`.
PS: sorry for asking to rename it earlier, i thought it's stored in a deeper context path instead of in (almost) the root of the context where ofc `errorType` makes no sense at all.